test/metabase/util/honeysql_extensions_test.clj

```diff
@@ -192,6 +192,7 @@
     typed-form     "date"   false
     typed-form     nil      false
     typed-form     #"tex.*" true
+    nil            #"tex.*" false
     typed-form     #"int*"  false
```

Comment:
If database-type is nilable, there should be a test for this here

Reply:
isn't the test on line 193 is testing with `nil` database-type?

or you mean `(is-of-type? nil nil)`?

Reply:
Ah, duh. Never mind

Reply:
either way I'll add the `nil nil` case to the test.

Reply:
Perhaps `is-of-type?` could be refactored to make the result of `nil` database-type more obvious. e.g. turning the `if` into a `cond`